### PR TITLE
Update one seller fee basis points

### PR DESCRIPTION
**Pull request by the-mercenaries-ltd**

### CODE REVIEW

File: src/update_metadata.rs [hunk 1]
```diff
@@ -25,6 +25,32 @@ use crate::limiter::create_rate_limiter;
 use crate::parse::{convert_local_to_remote_data, parse_cli_creators, parse_keypair};
 use crate::{constants::*, parse::parse_solana_config};
 
+pub fn update_seller_fee_basis_points_one(
+    client: &RpcClient,
+    keypair: Option<String>,
+    mint_account: &str,
+    new_seller_fee_basis_points: &u16,
```
Comment (samuelvanderwaal) [inline src/update_metadata.rs:32]: Nit: this doesn't need to be a reference as Rust primitives are stack copied when needed.

**Comment by samuelvanderwaal** [comment]
This looks good, thanks! Run `rust fmt` on it to fix the CI issue and I'll give it a quick test and merge it.

File: src/update_metadata.rs (hunk 1 above)
Comment (the-mercenaries-ltd) [reply src/update_metadata.rs:32]: Sorry, I don't understand. Are you saying don't use & in function parameters?

**Comment by the-mercenaries-ltd** [comment]
Happy to help 👍

File: src/update_metadata.rs (hunk 1 above)
Comment (samuelvanderwaal) [reply src/update_metadata.rs:32]: Yeah the `&` shouldn't be necessary, but it's not a big deal.

**Comment by samuelvanderwaal** [comment]
Merged! Thanks again!

**Pull request merged**